tst/PeerConnectionFunctionalityTest.cpp

```diff
@@ -106,6 +109,77 @@ TEST_F(PeerConnectionFunctionalityTest, connectTwoPeersForcedTURN)
     deinitializeSignalingClient();
 }
 
+// Assert that two PeerConnections can connect and then termintate one of them, the other one will eventually report disconnection
+TEST_F(PeerConnectionFunctionalityTest, connectTwoPeersThenDisconnectTest)
+{
+    if (!mAccessKeyIdSet) {
+        return;
+    }
+
+    RtcConfiguration configuration;
+    PRtcPeerConnection offerPc = NULL, answerPc = NULL;
+    UINT32 i;
+    RtcSessionDescriptionInit sdp;
+    BOOL connected = FALSE;
+    SIZE_T stateChangeCount[7];
```

Comment:
Can we determine the `7` at runtime? Just so it is a little more self documenting what this contains.

Reply:
added a new enum for counting the total number of states.